packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -121,9 +129,9 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
       maxExtent: itemExtent,
     );
 
-    final int firstIndex = getMinChildIndexForScrollOffset(scrollOffset, itemExtent);
+    final int firstIndex = getMinChildIndexForScrollOffset(scrollOffset, itemStride);
```

Comment:
The meaning of this seems to be a little more subtle than it was. If I'm understand the code correctly, it's no longer the index of the child that overlaps scrolloffset, but the first child+itemSpacing?